test/integration/acceptance/set_account_detail_test.cpp

```diff
@@ -65,6 +66,24 @@ TEST_F(SetAccountDetail, Self) {
 }
 
 /**
+ * C273
+ * @given a user with required permission
+ * @when execute tx with SetAccountDetail command with inexistent user
+ * @then there is no tx in proposal
+ */
+TEST_F(SetAccountDetail, NonExistentUser) {
+  IntegrationTestFramework(1)
+      .setInitialState(kAdminKeypair)
+      .sendTx(makeUserWithPerms({interface::permissions::Role::kSetDetail}))
+      .skipProposal()
+      .skipBlock()
+      .sendTxAwait(
+          complete(baseTx("inexistent@" + kDomain, kKey, kValue)),
```

Comment:
Please move all non-standard(default) values as const variables to the beginning of the test case. 
This is our best practice we are trying to follow.